DTS-STN/passport-status
# Updated Check Status info content

## [ADO-1576](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1576)

### Description
This PR updates the content for status messaging after a user submits the check status form. Since the content for each status is now vastly different, the structure had to be changed. This PR implements a component for each status which are controlled by the updated `StatusInfo` component (now named `CheckStatusInfo`). The Canada Post and Fedex statuses now also directly link to the tracking page with the user's tracking number already plugged in.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/status` and try entering mock data to see the different statuses.

### Additional Notes
As can be seen in a few of the tests, this PR does not include managing state to decide whether or not the phone number for the call center is shown, that will be added in a future PR.

In our mock data, we have a few entries with status code 3 or 4 (shipped Canada Post/Fedex) that don't have manifest numbers. Obviously this makes our content show up with blanks which is not ideal. Do we need to account for this and conditionally render the tracking number text, or will these codes always be accompanied by a tracking number going forward?
